e2e/tests/server.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
We are using a very basic setup of `express` here.

Do you mean using just `http-server` for instance and launching it directly from command line? It would just move the "complexity" to the Makefile instead of this script IMO. :)